lib/src/chain.dart

```diff
@@ -39,6 +39,9 @@ final _specKey = new Object();
 ///             "$stackChain");
 ///     });
 class Chain implements StackTrace {
+  /// The original stack trace
```

Comment:
"The original stack traces from which the components of this chain were parsed."

You should also mention that this may contain `null` elements for traces that don't have originals available.